robottelo/ui/locators/menu.py

```diff
@@ -4,6 +4,16 @@
 from selenium.webdriver.common.by import By
 from .model import LocatorDict
 
+# This is the old menu container xpath, was failing in upstream
```

Comment:
I would keep away from 'upstream' term in the code as it is too application specific